src/hooks/index.js

```diff
@@ -551,6 +551,13 @@ exports.sendEmail = (context, maildata) => {
 	const userIds = (typeof maildata.userIds === 'string' ? [maildata.userIds] : maildata.userIds) || [];
 	const receipients = [];
 
+	// create email attachments
+	const files = maildata.attachments;
+	const attachments = [];
+	files.forEach((element) => {
+		attachments.push({ filename: element.name, content: new Buffer(element.data) });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [new Buffer() is deprecated. Use Buffer.from(), Buffer.alloc(), or Buffer.allocUnsafe() instead. (no-buffer-constructor)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4570602)